src/utils/interpret-svg.js

```diff
@@ -343,15 +383,45 @@ var read = {
   },
 
   defs: function(node) {
-    var error = new TwoError('interpret <defs /> not supported.');
-    console.warn(error.name, error.message);
     return null;
   },
 
-  use: function(node) {
-    var error = new TwoError('interpret <use /> not supported.');
-    console.warn(error.name, error.message);
-    return null;
+  use: function(node, styles) {
+    var href = node.getAttribute('href') || node.getAttribute('xlink:href');
+    if (!href) {
+      var error = new TwoError('encountered <use /> with no href.');
+      console.warn(error.name, error.message);
+      return null;
+    }
+
+    var template = read.defsCache.get(href.slice(1));
+    if (!template) {
+      var error = new TwoError(
+        'unable to find element for reference ' + href + '.'
+      );
+      console.warn(error.name, error.message);
+      return null;
+    }
+    var fullNode = template.cloneNode(true);
+
+    var overwriteAttrs = new Set(
```

Comment:
I'm not sure if `Set` is supported on the range of browsers that Two.js supports (still working this out in #454)

Reply:
I switched to a regular object instead.